src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
```

Comment:
WorkflowOptions is constructed with a String -> String map (usually as a JsObject).

During construction, any keys in the map that are in the `workflow-options.encrypted-fields` list will be replaced with a JSON object which represents the encrypted value.  For example, if the initial map is

```json
{
    "encrypt_this": "foo",
    "dont_encrypt": "bar"
}
```

Then upon construction, if `workflow-options.encrypted-fields=["encrypt_this"]`, then the resulting JSON object is:

```json
{
    "encrypt_this": {
        "iv": "... base64 encoded initialization vector ..."
        "ciphertext": "... base64 encoded cipher text" ..."
    },
    "dont_encrypt": "bar"
}
```

When the client calls `workflowOptions.get("encrypt_this")`, the `get` method will recognize that the value is encrypted and decrypt it before sending back the unencrypted value.